pallets/capacity/src/lib.rs

```diff
@@ -108,11 +119,17 @@ pub mod pallet {
 		/// The overarching event type.
 		type RuntimeEvent: From<Event<Self>> + IsType<<Self as frame_system::Config>::RuntimeEvent>;
 
+		/// The overarching freeze reason.
+		type RuntimeFreezeReason: From<FreezeReason>;
+
 		/// Weight information for extrinsics in this pallet.
 		type WeightInfo: WeightInfo;
 
-		/// Function that allows a balance to be locked.
-		type Currency: LockableCurrency<Self::AccountId, Moment = BlockNumberFor<Self>>;
+		/// Functions that allow a fungible balance to be changed or frozen.
+		type Currency: MutateFreeze<Self::AccountId, Id = Self::RuntimeFreezeReason>
```

Comment:
Updating Frequency's `type Currency` with the traits from `frame_support::traits::tokens::fungible`